content/developer/reference/cli.rst

```diff
@@ -314,9 +314,18 @@ Emails
 
 .. option:: --smtp-port <port>
 
-.. option:: --smtp-ssl
+.. option:: --smtp-ssl [encryption]
 
-    If set, odoo should use SSL/STARTSSL SMTP connections
+    Connection encryption, accepted values are:
```

Comment:
The "if set" start of the previous version was helping understand that this flag is not required.

```suggestion
    If specified, this flag indicates the connection encryption. Accepted values are:
```